t/024_boolean_header_fields.t

```diff
@@ -2,6 +2,10 @@ use Test::More;
 use strict;
 use warnings;
 
+plan skip_all => "this test has not been converted as there's public admin on the new server";
```

Comment:
I don't like that we're skipping this since it had been a problem in the past. We should probably get this working again soon.